### PR TITLE
Doc/30 Generate doc for new edml release

### LINKED ISSUE #30: Generate EDML documentation for new version (1.1.0)

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #30

### CODE REVIEW

File: .github/workflows/deploy_schema_documentation.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 name: Deploy Schema Documentation
-on:
-  push:
-    branches:
-      - master
+on: ["push"]
+  #push:
+  #  branches:
+#   - master
```
Comment (morazow) [inline .github/workflows/deploy_schema_documentation.yml:5]: Can these commented lines be removed? No need to check that it is the master branch?

File: .github/workflows/deploy_schema_documentation.yml [hunk 2]
```diff
@@ -15,7 +15,10 @@ jobs:
       - name: Install and Build
         run: |
           sudo npm install -g bootprint bootprint-json-schema
-          bootprint json-schema src/main/resources/schemas/edml-1.0.0.json doc/gp_pages/schema_doc
+          mkdir doc/gp_pages/schema_doc
```
Comment (morazow) [inline .github/workflows/deploy_schema_documentation.yml:18]: Could you please rename `gp_pages` to `gh_pages` if you meant Github pages?

**Review by morazow — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy_schema_documentation.yml (hunk 1 above)
Comment (jakobbraun) [reply .github/workflows/deploy_schema_documentation.yml]: Oh, I forgot to push the last commit that changed that back.

File: .github/workflows/deploy_schema_documentation.yml (hunk 2 above)
Comment (jakobbraun) [reply .github/workflows/deploy_schema_documentation.yml]: :+1:

**Review by morazow — APPROVED** [review]

**Pull request merged**